client/src/components/auth/Register.jsx

```diff
@@ -1,18 +1,100 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import Card from 'react-bootstrap/Card';
-import Button from 'react-bootstrap/Button';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Form, Button, Container, Alert } from 'react-bootstrap';
 
 const Register = () => {
+    // creates initial states
+    const [name, setName] = useState('');
+    const [username, setUsername] = useState('');
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [message, setMessage] = useState('');
+    // allows to use useNavigate to redirect to link
+    const navigate = useNavigate();
+
+    const handleRegister = async (e) => {
+        e.preventDefault();
+        try {
+            const response = await fetch('http://localhost:3001/register', {
```

Comment:
I tried using just /register, but I think I still need to include localhost:3001 to have the connection to the server. 

@davidmedina-contact 
Should I look more into this to change for this PR, or could this be something I submit changes closer to having a more production ready environment?

Reply:
Do you have a proxy set up in your vite.config file? I believe this is one way to solve this.

Reply:
I added an environmental variable to dynamically build the URLs when fetching from the database. After looking into it more, I believe this will be quite helpful when having to change environments.

Submitted changes with the commit:  [4858085](https://github.com/abbiehuynh/my-crochet-corner/pull/4/commits/48580857b35b08e2cc6f35968a95581aca393961)